<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-S2M2L4JO" failures="0" tests="1" name="testng_runner.TestNG_runner" time="0.272" errors="1" timestamp="2023-01-02T18:26:39 IST" skipped="0">
  <testcase classname="testng_runner.TestNG_runner" name="setUpClass" time="0.272">
    <error message="Failed to parse resource at: file:///C:/Selenium_Coding_Ducat/cucumberjavaselenium/featurefile/login.feature
(2:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;Features: Login Testcases set&#039;
(3:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;Background:Verify user must have valid crendentials&#039;
(4:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;Scenario: Users is able to login into Applications&#039;
(5:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;Given: Open Applications &amp; Enter URL&#039;
(6:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;When: User Enter the Valid Username&#039;
(7:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;And: User Enter the Valid Password&#039;
(8:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;And: user clicked on Login Button&#039;
(9:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &#039;Then: User in LOgged into Application Successfully&#039;
(10:0): unexpected end of file, expected: #TagLine, #FeatureLine, #Comment, #Empty" type="io.cucumber.core.gherkin.FeatureParserException">
      <![CDATA[io.cucumber.core.gherkin.FeatureParserException: Failed to parse resource at: file:///C:/Selenium_Coding_Ducat/cucumberjavaselenium/featurefile/login.feature
(2:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Features: Login Testcases set'
(3:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Background:Verify user must have valid crendentials'
(4:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Scenario: Users is able to login into Applications'
(5:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Given: Open Applications & Enter URL'
(6:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'When: User Enter the Valid Username'
(7:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'And: User Enter the Valid Password'
(8:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'And: user clicked on Login Button'
(9:1): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Then: User in LOgged into Application Successfully'
(10:0): unexpected end of file, expected: #TagLine, #FeatureLine, #Comment, #Empty
at io.cucumber.core.gherkin.messages.GherkinMessagesFeatureParser.parse(GherkinMessagesFeatureParser.java:53)
at io.cucumber.core.feature.FeatureParser.parseResource(FeatureParser.java:45)
at java.base/java.util.function.BiFunction.lambda$andThen$0(BiFunction.java:70)
at io.cucumber.core.resource.ResourceScanner.lambda$processResource$1(ResourceScanner.java:79)
at io.cucumber.core.resource.PathScanner$ResourceFileVisitor.visitFile(PathScanner.java:75)
at io.cucumber.core.resource.PathScanner$ResourceFileVisitor.visitFile(PathScanner.java:60)
at java.base/java.nio.file.Files.walkFileTree(Files.java:2812)
at io.cucumber.core.resource.PathScanner.findResourcesForPath(PathScanner.java:53)
at io.cucumber.core.resource.PathScanner.findResourcesForUri(PathScanner.java:31)
at io.cucumber.core.resource.ResourceScanner.findResourcesForUri(ResourceScanner.java:61)
at io.cucumber.core.resource.ResourceScanner.scanForResourcesUri(ResourceScanner.java:134)
at io.cucumber.core.runtime.FeaturePathFeatureSupplier.loadFeatures(FeaturePathFeatureSupplier.java:62)
at io.cucumber.core.runtime.FeaturePathFeatureSupplier.get(FeaturePathFeatureSupplier.java:45)
at io.cucumber.testng.TestNGCucumberRunner.<init>(TestNGCucumberRunner.java:122)
at io.cucumber.testng.AbstractTestNGCucumberTests.setUpClass(AbstractTestNGCucumberTests.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUpClass -->
  <system-out/>
</testsuite> <!-- testng_runner.TestNG_runner -->
